src/node_perf.cc

```diff
@@ -334,9 +334,9 @@ inline Local<Value> GetName(Local<Function> fn) {
 void TimerFunctionCall(const FunctionCallbackInfo<Value>& args) {
   Isolate* isolate = args.GetIsolate();
   HandleScope scope(isolate);
```

Comment:
This HandleScope looks unnecessary; API callbacks implicitly have one.

Reply:
Thanks, removed.